doc/source/user_guide/io.rst

```diff
@@ -1806,6 +1807,20 @@ The ``Series`` object also has a ``to_string`` method, but with only the ``buf``
 ``na_rep``, ``float_format`` arguments. There is also a ``length`` argument
 which, if set to ``True``, will additionally output the length of the Series.
 
+   **Formatting missing values**
+
+   Formatting functions passed to the ``formatters`` argument above, are not
+   called for *missing values*, *string data types*, or ``PandasObject``.
+
+   If you need to control the format of missing values, consider these options:
+
+   - To replace *all* missing values (e.g ``NaN``) in output with a given
+     string, use the ``na_rep`` argument above.
+
+   - To replace missing values on a **per-column** level, call ``fillnans()``
```

Comment:
`` :meth:`Series.fillna` ``, I think.

Reply:
Correct 🙂 
...Though Sphinx doesn't seem to automatically link that to [the relevant page](https://pandas.pydata.org/docs/reference/api/pandas.Series.fillna.html) (at least when I build locally), which might be confusing to a user working with a `DataFrame`. (Even though I gather DataFrame leverages `Series` / it's the more generic type).